index.html

```diff
@@ -1,14 +1,177 @@
+            <img src="img/icon/getsocial.png" alt="GET REWARDED" class="features__icon-content">
+          </div>
+          <h4 class="features__heading heading heading_4">GET SOSIAL</h4>
+          <hr class="features__heading-delimiter">
+          <p class="features__text text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse fringilla fringilla.
+          </p>
+        </li>
+      </ul>
+    </div>
+  </section>
+  <section class="updates">
+    <div class="updates__container container">
+      <ul class="updates__menu menu">
+        <li class="updates__menu-item">
+          <img src="img/bg/bgphone.png" alt="phone" class="updates__menu-img">
+        </li>
+        <li class="updates__menu-item ">
+          <div class="updates__item-content">
+            <h4 class="updates__heading heading heading_4">INTRODUCING THE FUDI APP</h4>
+            <hr class="updates__heading-delimiter">
+          </div>
+
+          <div class="updates__item-textcontent">
+            <p class="updates__text text">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.<br><br> Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.<br><br> Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+            </p>
+            <img src="img/icon/appstore.png" alt="appstore" class="updates__icon">
+            <img src="img/icon/googleplay.png" alt="googleplay" class="updates__icon">
+          </div>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+
+  <section class="reviews">
+    <div class="reviews__container container">
+      <div class="reviews__menu menu">
+        <div class="reviews__menu-item">
+          <div class="reviews__item-content">
+            <img src="img/icon/person.png " alt="person" class="reviews__icon">
+            <p class="reviews__text-quote text"><q>I am so happy because i found this recipe and it just made my life
+            easier.<br>
```

Comment:
do not use `<br>`s. Because when you do responsive markup your text won't flow naturally